radis/lbl/calc.py

```diff
@@ -685,11 +719,15 @@ def _is_at_equilibrium():
         drop_columns = "auto"
 
     # Run calculations
-    sf = SpectrumFactory(
+    if CustomSpectrumFactory:
+        sfclass = CustomSpectrumFactory
+    else:
+        sfclass = SpectrumFactory
```

Comment:
to me, there is no other need than specifying a custom Lorentzian broadening, and this need is perfectly fit by `lbfunc`